scripts/run_docker_build.sh

```diff
@@ -52,7 +52,7 @@ export LANG=en_US.UTF-8
 # These were specific to installing matplotlib. I really want to avoid doing this if possible, but in some cases it
 # is inevitable (without re-implementing a full OS), so I also really want to ensure we can annotate our recipes to
 # state the build dependencies at OS level, too.
-yum install -y libXext libXrender libSM tk libX11-devel
+yum install -y libXext libXrender libSM tk libX11-devel mesa-libGL-devel
```

Comment:
@pelson, would it be alright if we include OpenGL support on Linux? Both Mac and Windows already have this. However, Linux is a simple package away from having this support.

Just like we have X11 to support Qt, it would be nice to have OpenGL to support a variety of other graphics programs. Does this sound ok to you?